proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -38,7 +38,8 @@ message TxResponse {
   // Time of the previous block. For heights > 1, it's the weighted median of
   // the timestamps of the valid votes in the block.LastCommit. For height == 1,
   // it's genesis time.
-  string timestamp = 12;
+  string   timestamp                    = 12;
+  repeated tendermint.abci.Event events = 13 [(gogoproto.nullable) = false];
```

Comment:
Can you add a proto comment about the difference of this VS logs?

Also, we should add a `Since: cosmos-sdk 0.45`, see https://github.com/cosmos/cosmos-sdk/blob/878e3e8a4ae58ba68c0866ef2cd5f59935156c16/proto/cosmos/bank/v1beta1/bank.proto#L96-L98

Reply:
Yes of course.